resources/views/secretariat/evaluation-form/avg.blade.php

```diff
@@ -1,7 +1,7 @@
 <form method="POST" action="">
     @csrf
     <blockquote>
-        Ha még nem vagy lezárva minden tárgyból, ne írd be az átlagodat! (Ha lezárnak a beküldési határidő előtt, akkor be tudod majd írni akkor is.)
```

Comment:
I originally meant that they can edit the form as many times as they want before the deadline.
The data is connected to the form, so they won't be edit this after the deadline.

Reply:
Oh, I see... what about this? _"A beküldési határidőig bármikor beírhatod majd."_